src/tests/Dictionary/DictionaryModal.test.jsx

```diff
@@ -231,18 +240,18 @@ describe('Test suite for dictionary modal', () => {
       errors: {},
     });
 
-    await wrapper3.instance().onSubmit(preventDefault);
-    expect(wrapper3.instance().state.errors).toEqual({});
+    await wrapper.instance().onSubmit(preventDefault);
+    expect(wrapper.instance().state.errors).toEqual({});
   });
 
   it('it should handle search input values', () => {
     props.isEditingDictionary = false;
-    const wrapper2 = mount(<DictionaryModal {...props} />);
-    const spy = jest.spyOn(wrapper2.find('DictionaryModal').instance(), 'searchInputValues');
-    wrapper2.find('input#react-select-3-input')
+    wrapper = mount(<DictionaryModal {...props} />);
```

Comment:
Same as above

Reply:
I had refactored the structure of the tests to align with the coding convention